File: app/src/main/java/net/sunniwell/app/linktaro/nettv/bean/PagePlayVideoBean.java
package net.sunniwell.app.linktaro.nettv.bean;

public class PagePlayVideoBean {
    private String channelIconUrl;
    private String channelName;
    private String channelNo;
    private String mCurPlayType;
    private String mPlayUrl;
    private String mSeekFileName;
    private String mSeekName;

    public String getCurPlayType() {
        return this.mCurPlayType;
    }

    public void setCurPlayType(String curPlayType) {
        this.mCurPlayType = curPlayType;
    }

    public String getChannelName() {
        return this.channelName;
    }

    public void setChannelName(String channelName2) {
        this.channelName = channelName2;
    }

    public String getChannelNo() {
        return this.channelNo;
    }

    public void setChannelNo(String channelNo2) {
        this.channelNo = channelNo2;
    }

    public String getChannelIconUrl() {
        return this.channelIconUrl;
    }

    public void setChannelIconUrl(String channelIconUrl2) {
        this.channelIconUrl = channelIconUrl2;
    }

    public String getmSeekFileName() {
        return this.mSeekFileName;
    }

    public void setmSeekFileName(String mSeekFileName2) {
        this.mSeekFileName = mSeekFileName2;
    }

    public String getmSeekName() {
        return this.mSeekName;
    }

    public void setmSeekName(String mSeekName2) {
        this.mSeekName = mSeekName2;
    }

    public String getmPlayUrl() {
        return this.mPlayUrl;
    }

    public void setmPlayUrl(String mPlayUrl2) {
        this.mPlayUrl = mPlayUrl2;
    }
}

File: app/src/main/java/com/eliving/tv/bean/PagePlayVideoBean.kt
package com.eliving.tv.bean

class PagePlayVideoBean {
    var channelIconUrl: String? = null
    var channelName: String? = null
    var channelNo: String? = null
    var curPlayType: String? = null
    private var mPlayUrl: String? = null
    private var mSeekFileName: String? = null
    private var mSeekName: String? = null

    fun getmSeekFileName(): String? {
        return mSeekFileName
    }

    fun setmSeekFileName(mSeekFileName2: String?) {
        mSeekFileName = mSeekFileName2
    }

    fun getmSeekName(): String? {
        return mSeekName
    }

    fun setmSeekName(mSeekName2: String?) {
        mSeekName = mSeekName2
    }

    fun getmPlayUrl(): String? {
        return mPlayUrl
    }

    fun setmPlayUrl(mPlayUrl2: String?) {
        mPlayUrl = mPlayUrl2
    }
}
